packages/pigment-react/README.md

```diff
@@ -23,6 +24,15 @@ A zero-runtime CSS-in-JS library that extracts the colocated styles to their own
 
 Pigment CSS supports Next.js and Vite with support for more bundlers in future. You must install the corresponding plugin, as shown below.
 
+### Why choose Pigment CSS
+
+Thanks to recent advancements in CSS (like CSS variables and `color-mix()`), "traditional" CSS-in-JS solutions that process styles at runtime are no longer required for unlocking features like color transformations and theme variables which are necessary for maintaining a sophisticated design system.
+
+Pigment CSS addresses the needs of the modern React developer by providing a zero-runtime CSS-in-JS styling solution as a successor to tools like Emotion and styled-components.
+
+Compared to its predecessors it offers improved performance and DX, and it's compatible with React Server Components.
```

Comment:
```suggestion
Compared to its predecessors it offers improved DX and performance at runtime at the cost of increased build time. And it's also compatible with React Server Components.
```

WDYT about adding the degraded build time ?

Reply:
Added it with a slight rephrasing! 👍